include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.h

```diff
@@ -126,7 +126,7 @@ class TimeIntBDF : public TimeIntBDFBase<Number>
   void
   calculate_sum_alphai_ui_oif_substepping(VectorType & sum_alphai_ui,
                                           double const cfl,
-                                          double const cfl_oif);
+                                          double const cfl_oif) override final;
```

Comment:
I think `final` already implies `override`. Do you want to keep it for readability?

Reply:
You are probably right, we can skip `override` then. Do you agree @nfehn ?

Reply:
We should then probably write only `final`.

Reply:
Done in 4e16828